app/views/search/advanced/index.html.erb

```diff
@@ -14,38 +14,25 @@
 
           <div class="form-group-search row">
             <div class="col col-form-label-lg">
-              <%= label_tag 'City' %>
-              <%= text_field_tag :city, nil, placeholder: 'Search by City', class: "form-control" %>
-            </div>
-          </div>
-
-          <div class="form-group-search row">
-            <div class="col col-form-label-lg" >
-              <%= label_tag :state %>
-              <%= select_tag :state, options_for_select(us_states), {class: 'custom-select' }  %>
+              <%= label_tag 'Location' %>
+              <%= text_field_tag :location, nil, placeholder: 'Search by Location', class: "form-control" %>
             </div>
           </div>
 
           <div class="form-group-search row" id="age-select">
             <div class="col col-form-label-lg" >
               <%= label_tag :age %>
-              <%= select_tag :age, options_for_select((0..150), 100), {class: 'custom-select' } %>
+              <%= select_tag :age, options_for_select((0..150), { }), {class: 'custom-select' } %>
             </div>
           </div>
 
           <div class="form-group-search row" id="year-select">
             <div class="col col-form-label-lg" >
               <%= label_tag :year %>
-              <%= select_tag :year, options_for_select((1981..2020), 2020), {class: 'custom-select' } %>
+              <%= select_tag :year, options_for_select(([0, 1981, 1982, 1983, 1984, 1985, 1986, 1987, 1988, 1989, 1990, 1991, 1992, 1993, 1994, 1995, 1996, 1997, 1998, 1999, 2000, 2001, 2002, 2003, 2004, 2005, 2006, 2007, 2008, 2009, 2010, 2011, 2012, 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020]), { }), {class: 'custom-select' } %>
             </div>
           </div>
 
-          <div class="form-group-search row" id="source-select">
-            <div class="col col-form-label-lg" >
-              <%= label_tag :source %>
-              <%= select_tag :source, options_for_select(["ObituaryHub Community", "The New York Times"]), {class: 'custom-select' }  %>
-            </div>
-          </div><br>
```

Comment:
No more search by source for now!

Reply:
This is because we are displaying both on every search index page right?

Reply:
Partly, also the more I thought about it I realized they needed to be able to select more than one and then we would need a collection of checkboxes...and that's when I gave up (for this iteration at least)